# Rune.msg
## From CS to CV : Health points of all robots

# Message

uint16 foe_hero
uint16 foe_standard1
uint16 foe_standard2
uint16 foe_sentry

uint16 ally_hero
uint16 ally_standard1
uint16 ally_standard2
uint16 ally_sentry
